### DISCUSSION TITLE
Camera flicker

### DISCUSSION THREAD

**Discussion by JdeV987** [OrcaSlicer/OrcaSlicer#826] [General]
If I view Fluidd dashboard via my web browser, the camera works 100%
If I view it via Orca, it flickers all the time.

In Win10 it works 100%, the problem is with Linux Mint 21.1 with Cinnamon desktop.

Any idea why or how to fix it?

**Comment by spamferkel** [comment]
Can confirm flickering for Linux Mint 20.3 XFCE

**Comment by alanmakestuff** [comment]
Can also confirm flickering for Pop_OS 22.04 LTS
